src/api/bycon.test.js

```diff
@@ -0,0 +1,23 @@
+const { buildQueryParameters } = require("./bycon")
+
+test("build a query", () => {
+  expect(
+    buildQueryParameters({
+      datasetIds: ["arraymap", "dipg"],
+      assemblyId: "GRCh38",
+      includeDatasetResonses: "ALL",
+      requestType: "variantCNVrequest",
+      referenceName: "9",
+      variantType: "DEL",
+      start: "20000001",
+      end: "21967753-23000000",
+      referenceBases: "N",
+      alternateBases: "G",
+      bioontology: ["NCIT:C102872"],
+      materialtype: "EFO:0009656",
+      filters: "filter1"
```

Comment:
You can just use an ontology or reference here in `filters`, e.g. "PMID:19074898".